### PR TITLE
Supercronic

### PR DESCRIPTION
**Pull request by grantbevis**
This PR covers the changes needed to build our own supercronic binary and the changes to entry.sh

### CODE REVIEW

**Comment by grantbevis** [comment]
Tested:

Broken crontab:
image

Working crontab:
image

File: base/Dockerfile [hunk 1]
```diff
@@ -28,8 +33,9 @@ RUN apk add --update --no-cache \
     /var/cache/apk/* \
     /.cache
 
+COPY --from=gobuilder /go/bin/supercronic /usr/local/bin/
 COPY --chmod=755 entry.sh /entry.sh
-COPY --link requirements.txt /
```
Comment (grantbevis) [inline base/Dockerfile:32]: I'm not 100% sure if this needs to be put back @modem7 - Docker for Mac errors and wouldn't let me build

**Comment by modem7** [comment]
To use things like --link etc, make sure to add `# syntax = docker/dockerfile:latest` to the top of the Dockerfile as it's a buildkit instruction.

**Comment by grantbevis** [comment]
> To use things like --link etc, make sure to add `# syntax = docker/dockerfile:latest` to the top of the Dockerfile as it's a buildkit instruction.

TIL - I will be using that trick at work! `--link` is a very cool instruction thanks dude

This PR is ready for review, I've briefly tested locally but don't have the infrastructure to fully test

**Comment by modem7** [comment]
> > To use things like --link etc, make sure to add `# syntax = docker/dockerfile:latest` to the top of the Dockerfile as it's a buildkit instruction.
> 
> TIL - I will be using that trick at work! `--link` is a very cool instruction thanks dude
> 
> This PR is ready for review, I've briefly tested locally but don't have the infrastructure to fully test

Totally missed this. On Friday I'll be able to do some testing due to Christmas holidays. I'll put a calendar entry in for it.

For how I'm using Buildkit on my fork, have a look at https://github.com/modem7/docker-borgmatic/tree/master/base-fullbuild and https://github.com/modem7/docker-borgmatic/tree/master/base-dockercli - should give you some ideas of how I'm doing things!

**Review by modem7 — DISMISSED** [review]
@grantbevis Took a bit of time to build, but have successfully built for both ARM64 and AMD64 with no issues or modifications.

I'm sure at some point we'll figure out how to optimise further (possibly with a separate image just to deal with supercronic to save time later as it's rarely updated vs the rest) but for now, I'm happy to give a LTGTM.

My build command:
`docker buildx build --rm=true --compress --no-cache --build-arg BUILDKIT_INLINE_CACHE=1 --platform linux/amd64,linux/arm64 -t borg:test .`

**Comment by psi-4ward** [comment]
image

does not work for me

**Pull request merged**